### PR TITLE
78 test lrp

### PR DESCRIPTION
**Pull request by geek-yang**
Explore LRP algorithm on MNIST and geometric shapes data sets. 
(fix small typos in one deepliftshap notebook as well, changes are minor and only related to markdowns).

### CODE REVIEW

File: example_data/xai_method_study/SHAP/DeepLiftSHAP_geometric_shapes.ipynb [hunk 1]
```diff
@@ -62,15 +62,15 @@
...
-    "### Load data (Geometric shapes dataset)"
+    "### Load data (geometric shapes dataset)"
    ],
    "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": 3,
    "source": [
-    "# load binary MNIST from local\n",
```
Comment (cwmeijer) [inline example_data/xai_method_study/SHAP/DeepLiftSHAP_geometric_shapes.ipynb:73]: 👍

**Review by cwmeijer — CHANGES_REQUESTED** [review]
I posted some comments on the binary mnist dataset here:
https://app.reviewnb.com/dianna-ai/dianna-exploration/blob/78-test-LRP/example_data%2Fxai_method_study%2FLRP%2FLRP_mnist_binary.ipynb%2F/file

The only 2 things that I would like to be changed before approval are a conclusion text block for both notebooks. Could be very short, one line even. But I think that would be very valuable.

Other than that, it looks very good. Great job!

File: example_data/xai_method_study/LRP/LRP_geometric_shapes.ipynb [hunk 2]
```diff
@@ -0,0 +1,433 @@
+{
```
Comment (cwmeijer) [inline example_data/xai_method_study/LRP/LRP_geometric_shapes.ipynb:1]: <p>Again, please add some conclusion.</p><p>Also, very hard to see what we should learn from these heatmaps. This is also a result of the rather difficult dataset we created. I wouldn't know myself what pixels I would expect to be important.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna-exploration/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (geek-yang) [reply example_data/xai_method_study/LRP/LRP_geometric_shapes.ipynb:1]: I add a short conclusion in the notebook. Thanks for the comment!

File: example_data/xai_method_study/SHAP/DeepLiftSHAP_geometric_shapes.ipynb (hunk 1 above)
Comment (geek-yang) [reply example_data/xai_method_study/SHAP/DeepLiftSHAP_geometric_shapes.ipynb:73]: Thanks for your comments, Chris. Please find my response via the ReviewNB inside the notebook.

**Review by cwmeijer — APPROVED** [review]

**Pull request merged**